code/_globalvars/bitfields.dm

```diff
@@ -173,5 +173,10 @@ GLOBAL_LIST_INIT(bitfields, list(
 	"rad_flags" = list(
 		"RAD_PROTECT_CONTENTS" = RAD_PROTECT_CONTENTS,
 		"RAD_NO_CONTAMINATE" = RAD_NO_CONTAMINATE,
+	),
+	"disease_flags" = list (

```

Comment:
this is just a bonus, i didn't add these, i just noticed they were missing from this list